CRM/Contact/Form/Task/EmailTrait.php

```diff
@@ -55,19 +55,6 @@ trait CRM_Contact_Form_Task_EmailTrait {
    */
   public $_toContactIds = [];
 
-  /**
-   * Store only "cc" contact ids.
-   * @var array
-   */
-  public $_ccContactIds = [];
```

Comment:
unused (that's not how we roll now & would just be an enotice if there was an issue)

Reply:
![image](https://user-images.githubusercontent.com/336308/130748416-3f54811f-21f9-4f97-9d5b-e0194d145f4a.png)

(afterwards)